GSA/made-in-america-data
# Removing waiver MIA-256

How to test: 

In the new waivers-data.json, you should not see a waiver with the id of '6206bc47baa79dbcd0a3322a'. 

You will see them in the csv files but those don't matter as those are deleted on every pull. 

https://cm-jira.usa.gov/browse/MIA-256
